config/webpack.dev.config.js

```diff
@@ -111,6 +111,7 @@ module.exports = Merge.smart(commonConfig, {
       DATA_API_BASE_URL: 'http://localhost:8000',
       ECOMMERCE_BASE_URL: 'http://localhost:18130',
       LICENSE_MANAGER_BASE_URL: 'http://localhost:18170',
+      ENTERPRISE_LEARNER_PORTAL_HOSTNAME: 'http://localhost:8734',
```

Comment:
hmm, on naming. the name says hostname, but this is actually a full blown url.. should this be called `***_URL` instead?

Reply:
Gooood point, I should check with @iloveagent57 to be sure the env variable I have is what I'm expecting to have.

Reply:
CHANGE NEEDED: The env variable on stage/prod is *just* the hostname, it doesn't include the `http://`. We need to change this to account for it.

Reply:
➕